salt/cert/init.sls

```diff
@@ -2,7 +2,7 @@ include:
   - crypto
 
 {% set ip_addresses = [] -%}
-{% set extra_names = ["DNS: " + grains['fqdn']] -%}
+{% set extra_names = ["DNS: " + grains['caasp_fqdn'] ] -%}
```

Comment:
We can leave this as empty list if the `CN` is the `caasp_fqdn`. Overlooked it before with the `fqdn` case.